ui/frontend/editor/rust_monaco_def.ts

```diff
@@ -0,0 +1,142 @@
+export const config = {
```

Comment:
Does this mean that there's no upstream Rust configuration for Monaco?

Reply:
There is, but I'm not happy with it. Some can be solved in upstream (for example not auto closing ' in lifetimes, or more detailed grammar) and some of changes are related to difference of textmate and monarch tokens so won't be accepted in upstream.